resources/v2/02_alter_bookings.sql

```diff
@@ -0,0 +1 @@
+  ALTER TABLE bookings MODIFY id INT UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=1;
```

Comment:
What is this for? By default, the starting value for `AUTO_INCREMENT` is 1 https://www.w3schools.com/sql/sql_autoincrement.asp